index.html

```diff
@@ -366,7 +367,7 @@ <h5>
         </h5>
 
         <figure id="fig-loose-setting">
-          <span style="text-align:center;"><img src="images/en/increased-inter-character-spacing.svg" alt="Examples of loose setting in horizontal writing mode." width="600" height="209"></span>
+          <span style="text-align:center;"><img class="invert-on-dark-mode" src="images/en/increased-inter-character-spacing.svg" alt="Examples of loose setting in horizontal writing mode." width="600" height="209"></span>
```

Comment:
Our theme colour, green, will be missing if we invert the images. We'll discuss if we need to fix this in the teleconference.